packages/kbn-i18n/tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "incremental": false,
-    "outDir": "./target/types",
-    "emitDeclarationOnly": true,
+    "allowJs": true,
```

Comment:
Do we need this setting?

Reply:
yeah we need in order to copy the `locales.js` file into the final target. We need to do it as part of the typescript compilation as Bazel don't allow us to change the outputs from a given target afterwards 😃

Reply:
will Bazel throw if there settings are out of sync with https://github.com/elastic/kibana/pull/99390/files#diff-e9b1a65072eccd923dcd9a253855a4f3abe1f6df6fd170ecc7f86638feac859dR86 ?

Reply:
@mistic makes sense! Thanks for pointing that out! Hopefully, we'll eventually remove this file when we tackle the action point in https://github.com/elastic/kibana/issues/88678

Reply:
@mshustov Bazel will throw if tsconfig.json settings are out of sync when comparing to what is declared in the bazel file